README.md

```diff
@@ -175,6 +175,28 @@ Dependencies: [openssl](https://www.openssl.org/)
 show-certificate google.com
 ```
 
+### :camera: Optimize images size
+Script: [optimage](optimage)
+Dependencies:
+```
+apt-get update && apt-get -y install gcc libc6-dev libjpeg-progs trimage imagemagick pngquant
```

Comment:
Hi, these installation instructions seem to be Ubuntu based.
Can you instead provide links to dependencies so that users can install them on their system on their own (if it's sufficiently feasible)? Thanks.